extensions.json

```
@@ -421,8 +421,8 @@
     {
       "id": "GitHub.vscode-pull-request-github",
       "repository": "https://github.com/microsoft/vscode-pull-request-github",
```

I notice that this extension made a pre-packaged `.vsix` version available.

In these cases, we prefer re-publishing official `.vsix` files rather than trying to re-package it ourselves (bears the risk of doing something wrong and uploading a broken extension to OpenVSX, e.g. if we forget an important packaging step).

```suggestion
      "download": "https://github.com/microsoft/vscode-pull-request-github/releases/download/0.20.0/vscode-pull-request-github-0.20.0.vsix",
```